examples/example-database.php

```diff
@@ -29,39 +30,27 @@
             [5] => ..
             [6] => 50
         )
-
 )
 */
-
 // Now we can build our query that loads data based on current page
 $start = ($data['currentPage']-1) * intval($elementsPerPage);
 $limit = intval($elementsPerPage); // MUCH SECURE
-$data_query = mysql_query("
-    SELECT *
-    FROM `my_table`
-    ORDER BY `id`
-    LIMIT {$start}, {$limit}
-");
-
+$db->prepare("SELECT *FROM `my_table`ORDER BY `id`LIMIT {$start}, {$limit}")
```

Comment:
If you are using prepared statements then please don't mix with hardcoded values but use "?" instead ?

Reply:
prevedi to sad na srpski